tests/sentry/grouping/grouping_inputs/frame-empty-list.json

```diff
@@ -0,0 +1,7 @@
+{
+    "event_id": "10a940c671e347d5861c7296b490aa63",
+    "platform": "csharp",
+    "stacktrace": {
+      "frames": []
+    }
+}
```

Comment:
Nice minimal test input! I assume that without the `and components` condition, this reproduces the crash?

Reply:
yep, and only `mobile` is failing